docs-2.0/nebula-cloud/2.how-to-create-subsciption.md

```diff
@@ -0,0 +1,31 @@
+# Create a subscription
+
+Before using Nebula Graph Cloud, you need to create a subscription on Azure. This topic describes how to create a subscription on Azure Marketplace.
+
+## Subscription workflow
+
+1. Enter the [Azure Marketplace](https://portal.azure.com/?l=en.en-us#blade/Microsoft_Azure_Marketplace/GalleryMenuBlade/selectedMenuItemId/home), and search for Nebula Graph Cloud in the search bar in Marketplace, or directly click [Nebula Graph Cloud]() to enter the subscription page. [TODO]
+
+2. Select a plan according to your own needs and click `Set up + subscribe`.
+
+   1. In the `Basics` page of **Subscribe Nebula Graph Cloud**, fill in the following plan details:
```

Comment:
If you are talking about something that you are able to view on a page, you will use "ON". So we'd better use On the xxx page, xxx